tools/api-markdown-documenter/src/utilities/ApiItemUtilities.ts

```diff
@@ -443,3 +442,25 @@ export function getModifiers(apiItem: ApiItem, modifiersToOmit?: ApiModifier[]):
 
 	return modifiers;
 }
+
+/**
+ * Generates a concise signature for a function.  Example: "getArea(width, height)"
+ */
+export function getConciseSignature(apiItem: ApiItem): string {
+	if (ApiParameterListMixin.isBaseClassOf(apiItem)) {
+		return `${apiItem.displayName}(${apiItem.parameters.map((x) => x.name).join(", ")})`;
+	}
+	return apiItem.displayName;
+}
+
+/**
+ * Converts bad filename characters to underscores.
+ */
+export function getSafeFilenameForName(apiItemName: string): string {
+	// eslint-disable-next-line unicorn/better-regex, no-useless-escape
+	const badFilenameCharsRegExp: RegExp = /[^a-z0-9_\-\.]/gi;
+
+	// TODO: This can introduce naming collisions.
+	// Will be fixed as part of https://github.com/microsoft/rushstack/issues/1308
```

Comment:
Do we want to keep this?

Reply:
Yeah, I would like to. If/when this gets resolved on the api-extractor side, it would be nice to clean things up here.

Reply:
I'll update the note though.
```suggestion
	// Note: This can introduce naming collisions.
	// TODO: once the following issue has been resolved in api-extractor, we may be able to clean this up:
	// https://github.com/microsoft/rushstack/issues/1308
```